### PR TITLE
Extending documentation to cover PAS 

### PR DESCRIPTION
**Pull request by galarragas**
Adding details on Android specific cases, in particular to cover PAS

### CODE REVIEW

File: bidding_auction_services_onboarding_self_serve_guide.md [hunk 1]
```diff
@@ -46,12 +46,24 @@ for [enrolling with Coordinators][14].
 
 ### Buyer
 
-  * Refer to [spec for DSP][28].
-  * Develop [generateBid][29]() for bidding.
-  * Develop [reportWin][30]() for event level reporting.
-  * Setup [buyer's Key/Value service][31].
-  * If your Key/Value server supports filtering of interest groups, refer to this
-    section and [metadata forwarding][25].
+Buyers could configure B&A to support PA (both for Chrome and Android traffic) and/or [PAS][120]
```
Comment (chatterjee-priyanka) [inline bidding_auction_services_onboarding_self_serve_guide.md:49]: Would it be possible to expand Protected Audience (PA) and continue using PA thereafter? PA acronym is not expanded before this point.

(Background / Context: When the self-serve explainer was published PAS was not public / explainer was not published. Hence we couldn't fork PAS and PA. I do plan to do an small update in this explainer later explaining Protected Auctions.)

File: bidding_auction_services_onboarding_self_serve_guide.md [hunk 2]
```diff
@@ -1081,3 +1093,11 @@ Refer to related publications on [Github][84].
 [114]: https://github.com/privacysandbox/bidding-auction-servers/blob/main/production/deploy/gcp/terraform/environment/demo/README.md#using-the-demo-configuration
 [115]: https://github.com/privacysandbox/bidding-auction-servers/tree/b27547a55f20021eb91e1e61b0d2175b4aee02ea/production/deploy/gcp/terraform/services
 [116]: https://github.com/privacysandbox/protected-auction-services-docs/blob/main/bidding_auction_event_level_reporting.md#rationale-for-the-design-choices
+[117]: https://github.com/privacysandbox/protected-auction-services-docs/blob/main/bidding_auction_services_protected_app_signals.md#buyer-ba-services
+[118]: [https://github.com/privacysandbox/protected-auction-services-docs/blob/main/bidding_auction_services_protected_app_signals.md#generatebid-udf](https://github.com/privacysandbox/protected-auction-services-docs/blob/main/bidding_auction_services_protected_app_signals.md#preparedataforadretrieval-udf)https://github.com/privacysandbox/protected-auction-services-docs/blob/main/bidding_auction_services_protected_app_signals.md#preparedataforadretrieval-udf
```
Comment (chatterjee-priyanka) [inline bidding_auction_services_onboarding_self_serve_guide.md:1097]: I think [118]: can be just the following?

https://github.com/privacysandbox/protected-auction-services-docs/blob/main/bidding_auction_services_protected_app_signals.md#preparedataforadretrieval-udf

File: bidding_auction_services_onboarding_self_serve_guide.md (hunk 1 above)
Comment (galarragas) [reply bidding_auction_services_onboarding_self_serve_guide.md:49]: Done, I expanded PAS too. Thanks!

File: bidding_auction_services_onboarding_self_serve_guide.md (hunk 2 above)
Comment (galarragas) [reply bidding_auction_services_onboarding_self_serve_guide.md:1097]: Thanks!

**Review by chatterjee-priyanka — APPROVED** [review]

**Pull request merged**